Content.Server/Cargo/Systems/CargoSystem.Shuttle.cs

```diff
@@ -144,6 +153,49 @@ private void UpdateShuttleCargoConsoles(CargoShuttleComponent component)
         }
     }
 
+    private void UpdatePalletConsoleInterface(EntityUid uid, CargoPalletConsoleComponent component)
+    {
+        var bui = _uiSystem.GetUi(component.Owner, CargoPalletConsoleUiKey.Sale);
+        var gridUid = Transform(uid).GridUid;
+        if (gridUid == null)
```

Comment:
you can shorten this with a pattern match 
```suggestion
        if (Transform(uid).GridUid is EntityUid gridUid)
```